apps/vtex-my-subscriptions-3/react/components/Summary/Content.tsx

```diff
@@ -2,15 +2,42 @@ import React, { FunctionComponent } from 'react'
 import { FormattedNumber, FormattedMessage } from 'react-intl'
 import { Total } from 'vtex.subscriptions-graphql'
 import { TranslateTotalizer } from 'vtex.totalizer-translator'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
+  'summaryRow',
+  'summaryItems',
+  'summaryDiscounts',
+  'summaryShipping',
+  'summaryTax',
+  'summaryInterest',
+  'summaryChange',
+  'summaryOther',
+  'summaryTotal',
+]
 
 const SummaryContent: FunctionComponent<Props> = ({ totals, currencyCode }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const taxTotals = totals.filter((total) =>
+    total.id.toLowerCase().includes('tax')
+  )
+  const nonTaxTotals = totals.filter(
+    (total) => !total.id.toLowerCase().includes('tax')
+  )
```

Comment:
I think you could get `taxTotals` and `nonTaxTotals` in the same loop using the `reduce`. What do you think about that?

```
const { taxTotals, nonTaxTotals } = totals.reduce<Accumulator>(
  (acc, total) => {
    const filteredArray = total.id.toLowerCase().includes('tax')
      ? acc.taxTotals
      : acc.nonTaxTotals;
    filteredArray.push(total);
    return acc;
  },
  { taxTotals: [], nonTaxTotals: [] }
);
```

Reply:
Arthur is on vacation, and due to the timing of publishing this PR I suggest that this optimization be done in the next PR. cc @polishq